templates/aws/aws_cloudformation_template.yml

```diff
@@ -5,16 +5,28 @@ Description:
   CloudFormation template for LANSA Database regression Testing.
 
 Parameters:
-  LANSA_VERSION:
+  LANSAVERSION:
     Type: String
     Description: LANSA Version that needs to be test
-  AMI_ID:
+  AMIID:
     Type: String
-    Description: AMI_ID for provisioning the VM    
-  MYSQL_SNAPSHOT_ARN:
+    Description: AMI_ID for provisioning the VM
+  SECURITYGROUPID:
+    Type: AWS::SSM::Parameter::Value<String>
+    Default: DB_REGRESSION_TESTING_VM_SG_ID
+    Description: Security Group to be attached to the VM.
+  SUBNETIDFORVM:
+    Type: AWS::SSM::Parameter::Value<String>
+    Default: DB_REGRESSION_TESTING_VM_SUBNET_ID
+    Description: Subnet ID where VM would be provisioned.
+  IAMROLEFORVM:
+    Type: AWS::SSM::Parameter::Value<String>
+    Default: DB_REGRESSION_TESTING_VM_IAM_PROFILE
+    Description: IAM Instance profile to be attached to VM.
```

Comment:
Specify a variable that is set to the IAM Role that the VM is currently using.

Reply:
It is using parameter store to fetch the IAM instance profile name.